### PR TITLE
NO MERGE: Connect to backend locally

### PR DESCRIPTION
**Pull request by dstuck**
Local integration of frontend with gql service using data pulled by scraper

Just pulled a couple 1000 institutions. Seems like all were late which may be because they're the oldest studies or maybe an issue to resolve in the gql store

image

### CODE REVIEW

File: src/components/InstitutionsTable.tsx [hunk 1]
```diff
@@ -66,10 +72,12 @@ function Institutions() {
   if (!data) {
     return <p>No data</p>;
   }
-  const institutions = data.institutions.map((ins, index) => ({
+  const institutions = data.allInstitutions.nodes.map((ins, index) => ({
     ...ins,
     key: index,
-  }));
+  })).sort(
+    (a,b) => (a.readyForReportCount > b.readyForReportCount) ? -1 : (a.readyForReportCount < b.readyForReportCount) ? 1 : 0
```
Comment (dstuck) [inline src/components/InstitutionsTable.tsx]: Temporarily added this `sort` just so I could see rows with actual results

**Comment by tomquisel** [comment]
Awesome! I'll leave it as a PR since it's just connecting to `0.0.0.0` right now.

**Comment by dstuck** [comment]
Oh yeah, accidentally didn't leave as a draft

**Comment by dstuck** [comment]
Gonna drop this for now since we have an actual backend!!

**Pull request closed without merging**